clippy_lints/src/utils/conf.rs

```diff
@@ -219,7 +219,8 @@ define_Conf! {
     ///
     /// #### Noteworthy
     ///
-    /// A type, say `SomeType`, listed in this configuration has the same behavior of `["SomeType" , "*"], ["*", "SomeType"]` in `arithmetic_side_effects_allowed_binary`.
+    /// A type, say `SomeType`, listed in this configuration has the same behavior of
+    /// `["SomeType" , "*"], ["*", "SomeType"]` in `arithmetic_side_effects_allowed_binary`.
```

Comment:
The reason for this change is to fix a `> 120 characters line` lint error on the markdown output.

Reply:
That's totally fine! The documentation should have been wrapped anyways. This is probably an artifact from the time, where the documentation was limited to a single line ^^